study1/week1/third-maximum-number.java

```diff
@@ -0,0 +1,28 @@
+// https://leetcode.com/problems/third-maximum-number/
+class Solution {
+    // Time Complexity: O(n), n: length of nums/ Runtime: 3ms
```

Comment:
The implementation of Java's `Array.sort(nums)` uses [quicksort](https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#sort-int:A-) thus the dominating factor of this function is `O(nlogn)`, as comparison based sorting cannot be faster than `O(nlogn)`